seata-spring-boot-starter/src/main/java/io/seata/spring/boot/autoconfigure/properties/registry/RegistryRedisProperties.java

```diff
@@ -27,7 +27,11 @@
 @ConfigurationProperties(prefix = REGISTRY_REDIS_PREFIX)
 public class RegistryRedisProperties {
     private String serverAddr = "localhost:6379";
-    private String db = "0";
+    private int db = 0;
```

Comment:
@guojingyinan219 Can you adapt other attributes of redis?
![image](https://user-images.githubusercontent.com/8758457/72313007-e3b4fa80-36c4-11ea-8dfc-c2279049389b.png)
This pr is also good, other attributes are not optional.